src/language/CSSUtils.js

```diff
@@ -83,19 +83,31 @@ define(function (require, exports, module) {
     
     /**
      * @private
+     * Returns state of a token
+     * @param {{editor:{CodeMirror}, pos:{ch:{string}, line:{number}}, token:{object}}} ctx
+     * @return {{tokenize:function, state:string, stateArg:string, context:Object}}
+     */
+    function _getState(ctx) {
```

Comment:
Can you rename it like _getTokenState or _getLocalState? _getState seems too common to me.
Can you add an assert for ctx?